pilot/pkg/model/service.go

```diff
@@ -145,6 +145,8 @@ const (
 	Passthrough
 	// DNSRoundRobinLB implies that the proxy will resolve a DNS address and forward to the resolved address
 	DNSRoundRobinLB
+	// Alias defines a Service that is an alias for another.
+	Alias
```

Comment:
is this only ever going to be used for k8s externalName services?  Or do you foresee any other uses?

Reply:
I hope this never extends, ExternalName is bad enough :-)